src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -38,4 +51,53 @@ private static String parseBankKeyFromId(final String itemId) {
 
         return itemId.split("-")[0];
     }
+
+    private static long parseLatestAcademicYear(final String academicYearString) {
+        // If this is a range date, pick the latest year
+        if (academicYearString.contains("-")) {
+            return Long.parseLong(academicYearString.split("-")[1]);
```

Comment:
Can we add this as a Java comment.  Not that @jtreuting told you but explaining that the rule.